### DISCUSSION TITLE
Quarto Visual Editor gone in RStudio, how to get it back

### DISCUSSION THREAD

**Discussion by IceManTBM3** [quarto-dev/quarto-cli#5781] [Q&A]
### Description

I'm on Win11 and running RStudio Version 1.3.959

- was on Quarto 1.2
- was on RStudio 1.2
- downloaded the Win Q 1.3 msi
- ran Quarto I.3 install from file manager
- launched RStudio 1.2, Visual Editor bar is still there, and I can create a Quarto Doc from the menu
- Run RStudio Upgrade to 1.3
- launch RStudio and the Visual Editor is not there...
- I have a few times uninstalled and re-installed via the msi to no avail..
- I can run Quarto commands in the terminal 

so what's the what I have out of sorts here? 

tmeek@AC-LT-TMEE-01  /c/Doc-Testing/diit-documentation/quarto (Tom_Doc_2)
$ quarto check
A new release of Deno is available: 1.28.2 → 1.34.1 Run `deno upgrade` to install it.

[>] Checking versions of quarto binary dependencies...
      Pandoc version 3.1.1: OK
      Dart Sass version 1.55.0: OK
[>] Checking versions of quarto dependencies......OK
[>] Checking Quarto installation......OK
      Version: 1.3.353
      Path: C:\Doc-Testing\diit-documentation\quarto\renv\library\R-4.2\x86_64-w64-mingw32\bin
      CodePage: 1252

[>] Checking basic markdown render....OK

[>] Checking Python 3 installation....OK
      Version: 3.9.7 (Conda)
      Path: C:/ProgramData/Anaconda3/python.exe
      Jupyter: 4.8.1
      Kernels: python3

[>] Checking Jupyter engine render....OK

[>] Checking R installation...........OK
      Version: 4.2.2
      Path: C:/Users/tmeek/AppData/Local/Programs/R/R-4.2.2
      LibPaths:
        - C:/Doc-Testing/diit-documentation/quarto/renv/library/R-4.2/x86_64-w64-mingw32
        - C:/Users/tmeek/AppData/Local/Temp/RtmpUHFIYm/renv-system-library
      knitr: 1.39
      rmarkdown: 2.14

![image](https://github.com/quarto-dev/quarto-cli/assets/129208946/21b8f824-4617-402d-bdc4-a2dd1f8e004c)

**Comment by mcanouil** [comment]
Thank you for reporting a possible bug, unfortunately this is an issue with the IDE, not Quarto CLI.
Please report this in <https://github.com/rstudio/rstudio>.
